examples/15_text_to_sql.py

```diff
@@ -0,0 +1,163 @@
+"""
+This example demonstrates how to convert natural language questions to SQL queries.
+It uses a sample e-commerce database schema and shows how to generate safe and efficient SQL queries.
+
+Like example 14 (templated instructions), this example shows how to use variables in the agent's
+instructions. The template variables ({{ db_schema }} and {{ question }}) are automatically populated
+from the input model's fields, allowing the instructions to adapt based on the input.
+
+The example includes:
+1. Simple SELECT query with conditions
+2. JOIN query with aggregation
+3. Complex query with multiple JOINs, grouping, and ordering
+"""
+
+import asyncio
+from typing import List
```

Comment:
this is deprecated so typing will fail after merging main into the branch (since we updated the lint tools on main)

Reply:
use `list` instead of `List` (no need to import)